### PR TITLE
fix: error not assignable to type 'StyleValue' in UiTextarea

### PR DESCRIPTION
**Pull request by pspaczek**
## Description
Bump `vue-tsc` and solve error in `UiTextarea`

## Related Issue

Closes #

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/components/atoms/UiTextarea/UiTextarea.vue
Comment (kubajmarek) [inline src/components/atoms/UiTextarea/UiTextarea.vue:1]: I don't love it. I liked the previous solution more, where we didn't assign any unnecessary values.

Wouldn't it work if we just replaced `null` with `undefined`? From what I can see only `null` was the problem, but `undefined` would be accepted and treated as no value.

Comment (pspaczek) [reply src/components/atoms/UiTextarea/UiTextarea.vue:1]: `undefined` used

File: src/components/atoms/UiTextarea/UiTextarea.vue [hunk 1]
```diff
@@ -136,9 +136,9 @@ const setTextareaSize = (mutationList: MutationRecord[]) => {
   const calcMinHeightPx = Number(minHeight.replace('px', '')) + (2 * Number(textareaMargin.replace('px', '')));
   const calcHeight = calcHeightPx > calcMinHeightPx ? null : `${calcMinHeightPx}px`;
 
-  textareaSize.width = width;
-  textareaSize.height = calcHeight;
-  textareaSize.minHeight = minHeight;
+  textareaSize.width = width || undefined;
+  textareaSize.height = calcHeight || undefined;
+  textareaSize.minHeight = minHeight || undefined;
```
Comment (AlanLes) [inline src/components/atoms/UiTextarea/UiTextarea.vue:141]: Is this necessary? What is the value of `width`, `calcHeight`, and `minHeight` if it is not in the `style`? Isn't it `undefined` indeed?

**Review by AlanLes — COMMENTED** [review]
I have one doubt, but overall looks good.

File: src/components/atoms/UiTextarea/UiTextarea.vue (hunk 1 above)
Comment (pspaczek) [reply src/components/atoms/UiTextarea/UiTextarea.vue]: TBH they return an empty string 🙈 :thank_you:

File: src/components/atoms/UiTextarea/UiTextarea.vue [hunk 2]
```diff
@@ -119,22 +119,23 @@ const resizeValue = computed(() => {
...
+  width: undefined,
+  height: undefined,
+  minHeight: undefined,
 });
 
 const textareaMargin = '2px';
 
 const setTextareaSize = (mutationList: MutationRecord[]) => {
   const { style } = mutationList[0].target as HTMLElement;
+  console.log(style);
```
Comment (DanielHajduk98) [inline src/components/atoms/UiTextarea/UiTextarea.vue:131]: console.log("missed console log?")

Comment (pspaczek) [reply src/components/atoms/UiTextarea/UiTextarea.vue]: fixed, thanks

File: src/components/atoms/UiTextarea/UiTextarea.vue [hunk 3]
```diff
@@ -119,9 +119,9 @@ const resizeValue = computed(() => {
 });
 const textarea = ref<HTMLTextAreaElement | null>(null);
 const textareaSize:Size = reactive({
-  width: null,
-  height: null,
-  minHeight: null,
+  width: '',
```
Comment (pspaczek) [inline src/components/atoms/UiTextarea/UiTextarea.vue:122]: empty strigs works well, then I use it instead undefined

**Review by DanielHajduk98 — APPROVED** [review]

**Pull request merged**